pkg/runtime/engine/config/config.go

```diff
@@ -5,13 +5,46 @@
 
 package config
 
+import (
+	"encoding/json"
+
+	"github.com/sylabs/singularity/pkg/plugin"
+)
+
 // Common provides the basis for all engine configs. Anything that can not be
 // properly described through the OCI config can be stored as a generic JSON []byte.
 type Common struct {
 	EngineName  string `json:"engineName"`
 	ContainerID string `json:"containerID"`
 	// EngineConfig is the raw JSON representation of the Engine's underlying config.
 	EngineConfig EngineConfig `json:"engineConfig"`
+
+	// Plugin is the JSON raw representation of the plugin configurations.
+	Plugin map[string]json.RawMessage `json:"plugin"`
```

Comment:
Does `PluginConfig` make more sense here to indicate it's list of plugin configs, not plugins, especially as we have `EngineConfig` directly above?

Reply:
Done